Sources/Swift/Integrations/SessionReplay/SentryOnDemandReplay.swift

```diff
@@ -21,52 +21,50 @@ enum SentryOnDemandReplayError: Error {
     case cantReadVideoSize
 }
 
-@available(iOS 16.0, tvOS 16.0, *)
 @objcMembers
 class SentryOnDemandReplay: NSObject {
     private let _outputPath: String
-    private let _onDemandDispatchQueue: DispatchQueue
-    
-    private var _starttime = Date()
-    private var _frames = [SentryReplayFrame]()
     private var _currentPixelBuffer: SentryPixelBuffer?
+    private var _totalFrames = 0
     
+    var frames = [SentryReplayFrame]()
```

Comment:
`m`: When we expose this property, we need to make it thread-safe. We currently only access it in tests. Maybe we can find a way to test the code, so we don't have to expose it to avoid making it thread safe when accessing it.

Reply:
This is exposed for test reasons, I will look into some alternative.